rules/integrations/okta/credential_access_user_impersonation_access.toml

```
@@ -0,0 +1,44 @@
+[metadata]
+creation_date = "2020/08/19"
+maturity = "production"
+updated_date = "2021/07/20"
+integration = "okta"
+
+[rule]
+author = ["Elastic"]
+description = """
+A user has initiated a session impersonation granting them access to our environment with the permissions of the user
+they are impersonating. This would likely indicate Okta administrative access and should only ever occur if requested
+and expected.
+"""
+from = "now-30m"
+index = ["filebeat-*", "logs-okta*"]
+interval = "15m"
+language = "kuery"
+license = "Elastic License v2"
+name = "Okta User Session Impersonation"
+note = """## Config
+
+The Okta Fleet integration, Filebeat module, or similarly structured data is required to be compatible with this rule."""
+references = [
+    ""
+]
+risk_score = 99
+rule_id = "cdbebdc1-dc97-43c6-a538-f26a20c0a911"
+severity = "high"
```

```suggestion
severity = "critical"
```
In case that we need to adjust to match the risk score